packages/style-loader/README.md

```diff
@@ -156,14 +156,8 @@ module.exports = {
       },
       {
         loader: 'postcss-loader',
-        useId: 'postcss',
         options: {
-          config: {
-
-            // Where to look for config (postcss.config.js)
-            // https://github.com/postcss/postcss-loader#path
-            path: neutrino.options.root
-          }
+          plugins: [require('autoprefixer')]
```

Comment:
@eliperelman like I show here

Reply:
Ah OK, it wasn't totally clear to me. Thanks!